pkg/cmd/generate.go

```diff
@@ -82,11 +87,25 @@ func RunGenerate(cmd *cobra.Command, args []string) error {
 	log.Printf("trying fallback")
 
 	// fallback - generate a new filename and put the content inside
-	newFileName := path.Join("generated-by-copilot-ops", "generated-by-copilot-ops.yaml")
-	r.Filemap.Files = map[string]filemap.File{
-		newFileName: {Path: newFileName, Content: output, Tag: newFileName},
+
+	outputSplit := strings.Split(output, "\n\n +----------------------------------------------------------+ \n\n")
```

Comment:
hmm, maybe instead of relying on this magic hardcoded separator, we can just return output as an array from `GenerateCode()` and then decide how to treat the list of completions here in the command logic.

Reply:
Updated, not sure if it's what you're looking for but I changed the main GenerateCode function to return a string slice. That string slice is then concatenated only for printing.